### PR TITLE
Update SIG Docs teams

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR updates teams for SIG Docs.

@jimangel 👋 I'm happy to close this in deference to your own PR.

### CODE REVIEW

**Comment by Bradamant3** [comment]
I know I don't have merge/approve rights in this repo, so I'll just chime in:
/lgtm

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -42,7 +42,7 @@ aliases:
     - nikhita
   sig-docs-leads:
     - bradamant3
-    - jaredbhatti
+    - jimangel
```
Comment (nikhita) [inline OWNERS_ALIASES]: nit: can you move this to another commit since it updates the leads?

File: config/kubernetes/sig-docs/teams.yaml [hunk 2]
```diff
@@ -210,21 +206,22 @@ teams:
...
     - cody-clark
-    - jaredbhatti
+    - gochist
     - jimangel
     - kbarnard10
+    - kbhawkey
+    - makoscafee
     - mistyhacks
-    - Rajakavitha1
+    - rajakavitha1
```
Comment (nikhita) [inline config/kubernetes/sig-docs/teams.yaml]: Would prefer to keep this as `Rajakavitha1` since that's how the GitHub username is structured: https://github.com/Rajakavitha1
